fixtures/service.js

```diff
@@ -13,14 +13,31 @@ window.EXAMPLE_SERVICE = {
     }
   },
   "spec": {
-    "port": 5434,
-    "protocol": "TCP",
+    "ports": [
+      {
+        "name": "",
+        "protocol": "TCP",
+        "port": 5434,
+        "targetPort": 3306
+      },
+      {
+        "name": "two",
+        "protocol": "TCP",
+        "port": 5435,
+        "targetPort": 3307
+      },
+      {
+        "name": "three",
+        "protocol": "UDP",
+        "port": 5436,
+        "targetPort": 3308
+      }
+    ],
     "selector": {
       "name": "database"
     },
     "portalIP": "172.30.17.6",
-    "containerPort": 3306,
     "sessionAffinity": "None"
   },
   "status": {}
-};
\ No newline at end of file
+};
```

Comment:
Maybe add a fixture for a headless service... `window.EXAMPLE_HEADLESS_SERVICE = ...` that has no ports and a portalIP of "None", and wire that up in the index.html